sensor/common/centralcaps/centralcaps.go

```diff
@@ -0,0 +1,28 @@
+package centralcaps
+
+import (
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	centralCaps = set.NewSet[centralsensor.CentralCapability]()
+
+	centralCapsMutex sync.RWMutex
+)
+
+// Has returns if the Central has the given capability.
+func Has(cap centralsensor.CentralCapability) bool {
+	centralCapsMutex.RLock()
+	defer centralCapsMutex.RUnlock()
+	return centralCaps.Contains(cap)
+}
+
+// Set sets the capabilities of the connected Central.
+func Set(caps []centralsensor.CentralCapability) {
```

Comment:
do we ever expect this to update after it's written to the first time?

Reply:
I suppose it might with the efforts of reconnecting on Sensor disconnects. There's an effort to support re-establishing communication, in this case the sensor/central hello will be exchanged again and the caps will be written to again.